interpreter/chapter.txt

```diff
@@ -131,9 +137,10 @@ Remember that given valid instruction sets, without any changes to our interpret
         "numbers": [7, 5, 8] }
 ~~~~
 
-At this point, we can begin to see how this structure is extensible - we can add methods on the interpreter object that describe many more operations (as long as we have a compiler to hand us well-formed instruction sets).
+At this point, we can begin to see how this structure is extensible: we can add methods on the interpreter object that describe many more operations (as long as we have a compiler to hand us well-formed instruction sets).
+
+#### Variables
 
-### Variables
 Next let's add variables to our interpreter. Variables require four pieces: an instruction for storing the value of a variable, `STORE_NAME`; an instruction for retrieving it, `LOAD_NAME`; and a mapping from variable names to values. For now, we'll ignore namespaces and scoping, so we can store the variable mapping on the interpreter object itself. Finally, we'll have to make sure that `what_to_execute` has a list of the variable names, in addition to its list of constants.
```

Comment:
The text says "Variable require four pieces" but the follow list seems to only contain three things: 

- an instruction for storing,
- an instruction for retrieving, and
- a mapping from variable names to values.

Do we need one more thing on that list, or should we change it to "Variables require three pieces"?